app/assets/stylesheets/frontend/views/_how-gov-works.scss

```diff
@@ -275,14 +263,6 @@
   p.small-print {
     @include ig-core-14;
   }
-  h2.coalition {
-    border-color: black;
-  }
-  .coalition h3 {
```

Comment:
I think this was needed to maintain the styles which I commented are missing in a previous commit on headings.